public/app/features/variables/pickers/shared/VariableOptions.tsx

```diff
@@ -78,7 +78,7 @@ export class VariableOptions extends PureComponent<Props> {
         className={`${tag.selected ? 'variable-option-tag pointer selected' : 'variable-option-tag pointer'}`}
         onClick={this.onToggleTag(tag)}
       >
-        <span className="fa fa-fw variable-option-icon"></span>
```

Comment:
This works as expected also without `fa fa-fw` see [this dahsboard](http://localhost:3000/d/yBCC3aKGk/templating-dashboard-links-and-variables?orgId=1).